### PR TITLE
#34 - Improved Lyra code

### PR DESCRIPTION
**Pull request by SamSalvatico**
### Why?
To improve code style requested in issue #34 

### What?

Refactor *src/plugins/Lyra* code

### CODE REVIEW

File: src/plugins/lyra.ts [hunk 1]
```diff
@@ -1,50 +1,86 @@
...
-      : `${cfFolder.outDir.pathname}/search.db.json`;
-  await fs.writeFile(dbFile, JSON.stringify(db));
+  return db;
+}
+
+function cleanupEpisodes(episodes: Episode[]): DbEpisode[] {
+  const turndownService = new TurndownService();
+  return episodes.map((e) => ({
+    title: e.title,
+    description: turndownService
```
Comment (brainrepo) [inline src/plugins/lyra.ts:63]: NIT: can we export this conversion in a helper function?

**Review by brainrepo — COMMENTED** [review]
LGTM just a little nit

**Comment by brainrepo** [comment]
Ps. potresti aggiungere per favore il tuo nome nella lista dei contributor nella pagina speakers?

File: src/plugins/lyra.ts (hunk 1 above)
Comment (SamSalvatico) [reply src/plugins/lyra.ts:63]: Sorry @brainrepo, to avoid make unneccessary (and wrong) stuffs, do you mean moving it to utils?

Comment (brainrepo) [reply src/plugins/lyra.ts:63]: yes man ;)

Comment (brainrepo) [reply src/plugins/lyra.ts:63]: ty

**Review by brainrepo — APPROVED** [review]
cool going to merge

**Pull request merged**